Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml

```diff
@@ -0,0 +1,232 @@
+﻿- type: entity
+  name: blue slime
+  id: MobAdultSlimes
+  parent: SimpleMobBase
+  description: It looks so much like jelly. I wonder what it tastes like?
+  suffix: Good
+  components:
+  - type: HTN
+    rootTask: SimpleHostileCompound
+  - type: Sprite
+    drawdepth: Mobs
+    netsync: false
+    sprite: Mobs/Aliens/slimes.rsi
+    layers:
+    - map: [ "enum.DamageStateVisualLayers.Base" ]
+      state: blue_adult_slime
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeCircle
+        radius: 0.30
+      density: 80
+      mask:
+      - MobMask
+      layer:
+      - MobLayer
+  - type: MovementAlwaysTouching
+  - type: MobThresholds
+    thresholds:
+      0: Alive
+      150: Dead
```

Comment:
This has 50 more hp than a human? Seems unnecessarily strong considering they can beat normal people 1v1, especially with their high damage.

Reply:
Okay, I'll reduce the value to 110, but on average they are weaker than tarantulas, slower, and a huge amount of water damage.